README.rst

```diff
@@ -47,9 +55,41 @@ SciPy a try!
 
 For the installation instructions, see INSTALL.rst.txt_.
 
-We appreciate and welcome contributions. If you would like to take part in
-SciPy development, take a look at the file CONTRIBUTING.rst_.
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. If you are considering larger
+contributions to the source code, please contact us through the [mailing
```

Comment:
This sentence I don't quite like anymore. Usually GitHub is the right avenue, and larger bug fixes don't need discussion. Can you just link to http://scipy.github.io/devdocs/dev/hacking.html instead?